src/test/movieSlice.test.js

```diff
@@ -0,0 +1,36 @@
+import moviesSlice, { fetchMovies } from '../data/moviesSlice'
+import { moviesMock } from './movies.mocks'
+
+describe('MovieSlice test', () => {
+    
+    it('should set loading true while action is pending', () => {
+        const action = {type: fetchMovies.pending};
+        const initialState = moviesSlice.reducer(
+        { 
+            movies: [], fetchStatus: '',
+        }, action);
+        expect(action).toEqual({type: fetchMovies.pending})
```

Comment:
It should validate that the state has been updated rather than matches 'pending'